Sources/TuistGenerator/Templates/FontsTemplate.swift

```diff
@@ -10,7 +10,7 @@ extension SynthesizedResourceInterfaceTemplates {
     {% set fontType %}{{param.name}}FontConvertible{% endset %}
     #if os(macOS)
       import AppKit.NSFont
-    #elseif os(iOS) || os(tvOS) || os(watchOS)
+    #elseif os(iOS) || os(tvOS) || os(watchOS) || vision(OS)
```

Comment:
Shouldn't this be 
```suggestion
    #elseif os(iOS) || os(tvOS) || os(watchOS) || os(visionOS)
```
?

Reply:
Yes, 100%. Sorry about that, bad copy/paste